### PR TITLE
Refactor logic

### LINKED ISSUE #29: Refactor LogicManager and Logic classes

### LINKED ISSUE #48: Refactor LogicManager and Logic class

### PR DESCRIPTION
**Pull request by Ziyang-98**
Changed Logic, LogicManager and added new Prefixes in CliSyntax

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/CliSyntax.java [hunk 1]
```diff
@@ -11,4 +11,10 @@
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
```
Comment (lettuceman4) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java:13]: why don't u delete the old commands ah

**Review by lettuceman4 — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/parser/CliSyntax.java (hunk 1 above)
Comment (Ziyang-98) [reply src/main/java/seedu/address/logic/parser/CliSyntax.java:13]: so that nich and jordan can refer to it when implementing the commands

Comment (Ziyang-98) [reply src/main/java/seedu/address/logic/parser/CliSyntax.java:13]: for tags we reusing it

**Pull request merged**